packages/experience-builder-types/src/constants.ts

```diff
@@ -0,0 +1,72 @@
+import { InternalSDKMode, SchemaVersions } from './types'
+
+export const SCROLL_STATES = {
+  Start: 'scrollStart',
+  IsScrolling: 'isScrolling',
+  End: 'scrollEnd',
+}
```

Comment:
As written per DM, it would be great to expose a type based on this constant like this:
```
export type ScrollState = typeof SCROLL_STATES[keyof typeof SCROLL_STATES];
```

Reply:
Similar for the other object literals below. This way, we can replace the old & deprecated types with those new ones.

Reply:
(or is this supposed to be a separate PR afterwards to reduce code changes?)

Reply:
I added them in next commit, lost them previously because of merge conflicts in types.ts

I added them in types.ts of experience-builder-types package